users/updates/components/c7tZQf926zKq.php

```diff
@@ -35,11 +35,11 @@
 }
 
 $db->query("CREATE TABLE us_fingerprint_assets (
-  kFingerprintAssetID int(11) UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
-  fkFingerprintID int(11) NOT NULL,
-  IP_Address varchar(255) NOT NULL,
-  User_Browser varchar(255) NOT NULL,
-  User_OS varchar(255) NOT NULL
+  kFingerprintAssetID SERIAL PRIMARY KEY,
+  fkFingerprintID INT NOT NULL,
```

Comment:
Please remain syntactically consistent. If you are using `integer`, use it everywhere.

Reply:
I will change all of these later when changing Boolean to smallint.